### PR TITLE
feat: support for Express

### PR DESCRIPTION
**Pull request by ayusham001**
**Task 1 – Add a README.md file**
  - [x] Create a new README.md file inside the packages/express folder and copy in the contents from packages/nextjs/README.md. Adjust the wording to reference Express where needed, including updating the URLs.

**Task 2 – Write the documentation**
  - [x] Locate the manual-setup-express.mdx folder in /docs/_snippets/. Use this file to write your documentation, overwriting any contents of that file. Follow the structure of this Next.js example for reference when writing yours.
  - [x] Update the table at the bottom of the client-adaptors.mdx page with the framework you’ve worked on.

**Task 3 – Update the webapp onboarding**
  - [x] In the route.tsx page located in _app.orgs.$organizationSlug.projects.$projectParam.setup.express, replace the <FrameworkComingSoon/> component with your new onboarding steps. Follow the same structure as the Next.js example when creating your own. Ignore the very top step for adding this framework to a new project; just focus on the steps to add it to an existing project (so don’t include the <RadioGroup/> toggle selector at the top). Make sure you also include the useProjectSetupComplete(); hook at the top of the page – this triggers a particle effect when the new job is successfully created and redirects to the Jobs list 🥳.
  - [x] In the FrameworkSelector.tsx file, add “supported” to the framework you’ve worked on.

/claim #451

### CODE REVIEW

**Comment by ayusham001** [comment]
"Hi @samejr, please review my PR and let me know if you have any feedback :)

**Comment by ayusham001** [comment]
Hi @samejr, just checking in with you since it has been a while. Please revert back to this whenever you find it suitable, I will be looking forward to your reply.

**Comment by samejr** [comment]
Thanks @ayusham001 sorry for the delay. @matt-aitken is making some updates to our CLI to better support these new frameworks so he'll be checking out your PR very soon.

**Comment by ayusham001** [comment]
Thank you for the update, @samejr 
No problem about the delay.

File: docs/_snippets/manual-setup-express.mdx [hunk 1]
```diff
@@ -1 +1,234 @@
...
+
+2. Add the following code to your route file:
+
+```javascript
+const express = require('express');
+const { createExpressRoute } = require('@trigger.dev/express');
+const { client } = require('./trigger'); // Update the path as per your project structure
+
+const router = express.Router();
+const { POST } = createExpressRoute(client);
```
Comment (matt-aitken) [inline docs/_snippets/manual-setup-express.mdx:107]: There's no function called this so this code won't work

**Comment by matt-aitken** [comment]
The code in the doc won't work, it references a function that doesn't exist in our express package. Closing this

**Comment by ayusham001** [comment]
Hi @matt-aitken,
I apologize for any misunderstanding. I have reviewed the documentation and made the necessary updates in my branch. Could you kindly consider reopening this pull request for a review? Thank you!

**Pull request closed without merging**